mongo/datadog_checks/mongo/metrics.py

```diff
@@ -353,8 +353,8 @@
     'collection.transactions.ops': RATE,
     'collection.transactions.latency_avg': (GAUGE, 'collection.transactions.latency.avg'),
     # collection query exec stats
-    'collection.collectionScans.total': GAUGE,
-    'collection.collectionScans.nonTailable': GAUGE,
+    'collection.collectionScans.total': RATE,
+    'collection.collectionScans.nonTailable': RATE,
 }
```

Comment:
Should this match `metadata.csv`?

Reply:
when the metric type is rate, the agent calculates the diff between 2 consecutive collections and submitted as [gauge](https://docs.datadoghq.com/metrics/types/?tab=rate#submission).